src/onegov/fsi/models/course_attendee.py

```diff
@@ -152,7 +166,7 @@ def role(self) -> str:
         assert self.user is not None
         return self.user.role
 
-    @property
+    @hybrid_property
```

Comment:
Since I converted `email` to a `hybrid_property` the test `test_add_edit_external_attendee` fails with `TypeError: 'email' is an invalid keyword argument for CourseAttendee`.
If I revert back the test passed but the `search-postgres?q=martin` fails due to the missing hybrid property...

Reply:
It's very possible that `setter` has to be defined before the `expression`. Although I'm still not sure if the auto-generated `__init__` for SQLAlchemy will contain settable hybrid properties, it might not.

But this whole thing seems to be heading in very much the same direction as your first attempt at implementing the indexing. Maybe it would be more robust to store the computed es properties like `es_public` in the database at the same time we store the index.

This way we don't have to add all these new potentially subtly incorrect hybrid properties just to use them within a SQL query and are closer to the original elastic search implementation. If we want more of these things to happen online down the road, we can always improve it later.